apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/ShardPrunableZipHashCalculator.java

```diff
@@ -56,9 +53,9 @@ public ShardPrunableZipHashCalculator(Event<ChunkedFileOps> fileChangedEvent, De
         this.fileChangedEvent = fileChangedEvent;
     }
 
-    public void onTrimDone(@Observes @Async TrimData trimData) {
+/*    public void onTrimDone(@Observes @Async TrimData trimData) {
         tryRecalculatePrunableArchiveHashes(trimData.getPruningTime());
```

Comment:
what do think of removing this commented code? in case of recollection, we have git with history